iac/stacks/common.py

```diff
@@ -313,6 +313,48 @@ def __init__(
                 open=True,
             )
 
+        self.report_s3_user = iam.User(
```

Comment:
This looks fine but does creating the IAM user via CDK mean that if we make a change in the future that replaces this, users whose url was created using the old/destroyed IAM but haven't accessed it yet will be unable to access?

Reply:
The presigned urls have a lifetime and then expire. The default is 3600 seconds https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/generate_presigned_url.html. If something changes the user can generate another url and should get a new presigned url.